core-aam/index.html

```diff
@@ -5711,6 +5711,48 @@ <h4>Not Mapped</h4>
               >. Its absence not only indicates that the accessible object is not grabbed, but further defines it as not grab-able. These cases are marked as "Not mapped" without an asterisk.
             </p>
           </section>
+          <h4 id="ariaActions"><code>aria-actions</code></h4>
+          <table aria-labelledby="ariaActions">
+            <tbody>
+              <tr>
+                <th>ARIA Specification</th>
+                <td>
+                  <a class="property-reference" href="#aria-actions"><code>aria-actions</code></a>
+                </td>
+              </tr>
+              <tr>
+                <th>MSAA + IAccessible2</th>
+                <td>
+                  <span class="property"
+                    >Interface: <code>IAccessibleAction</code>, using the accessible nodes matching `aria-actions` IDREFs, if the referenced objects are in the accessibility tree</span
+                  ><br />
+                  <span class="property">Object Attribute: <code>aria-actions</code> &lt;value&gt;</span>
+                </td>
+              </tr>
+              <tr>
+                <th><abbr title="User Interface Automation">UIA</abbr></th>
+                <td>
+                  <span class="property"
+                    >Custom Property: <code>AccessibleActions</code>: element array of accessible nodes matching IDREFs, if the referenced objects are in the accessibility tree</span
```

Comment:
Custom properties in UIA require a GUID. I think that needs to be specified here.

A bigger problem is that according to the [UIA documentation](https://learn.microsoft.com/en-us/windows/win32/api/uiautomationcore/ns-uiautomationcore-uiautomationpropertyinfo#remarks), custom properties in UIA can return a single element, but *not* an element array. I'm not sure if this is a real limitation or a mismatch between the documentation and reality, but either way, this needs to be clarified with Microsoft.

Reply:
I checked with Doug Geoffray, and it looks like it's just an oversight in the documentation. I believe any of the listed types can also be used as an array type for a property value.

We should be good to go to create AccessibleActions as a `UIAutomationType_Element[]` array 👍